lib/test-runner.js

```diff
@@ -5,7 +5,7 @@
 // // testem.js
 // module.exports = {
 //   "launchers": {
-//     "Electron": require("ember-electron/lib/test-support/test-runner")
+//     "Electron": require("ember-electron/lib/test-runner")
```

Comment:
No objection but curious why `test-support/test-runner` was changed to `test-runner`. Just tidying up?

Reply:
Disregard -- just noticed notes in commit message
https://github.com/adopted-ember-addons/ember-electron/commit/e4a719b0e6e131fe4c02a110bd21840bc2932edf